config.json

```diff
@@ -1524,6 +1405,155 @@
                     }
                 }
             }
+        },        
+        "Config.ContactTab": {
+            "label": "Config.contactTab",
+            "formFields": {
+                "contact.shop_mail": {
+                    "type": "inputText",
+                    "required": false,
+                    "label": "Config.contactShopMailLabel",
+                    "options": {
+                        "defaultValue": "your@email.com"
+                    }
+                },
+                "contact.shop_mail_cc": {
+                    "type": "inputText",
+                    "required": false,
+                    "label": "Config.contactShopMailCcLabel",
+                    "options": {
+                        "defaultValue": "your2@email.com,your3@email.com"
+                    }
+                },
+                "contact.shop_mail_bcc": {
+                    "type": "inputText",
+                    "required": false,
+                    "label": "Config.contactShopMailBccLabel",
+                    "options": {
+                        "defaultValue": "your2@email.com,your3@email.com"
+                    }
+                },
+                "contact.extend_orderid_to_mail_subject": {
+                    "type": "checkBox",
+                    "required": false,
+                    "label": "Config.contactExtendOrderidToMailSubjectLabel",
+                    "options": {
+                        "defaultValue": "false"
+                    }
+                },
+                "contact.opening_times": {
```

Comment:
Zeile 1444 bis Zeile 1461 wurden in einem [anderen PR](https://github.com/plentymarkets/plugin-ceres/pull/1067) von mir entfernt, da deprecated. Diese Zeilen können gelöscht werden, da diese Texte aus der Mehrsprachigkeit geladen werden.